detect-configuration/src/main/java/com/synopsys/integration/detect/configuration/DetectProperty.java

```diff
@@ -83,6 +83,10 @@
     @HelpDescription("The API token used to authenticate with the Black Duck Server.")
     BLACKDUCK_API_TOKEN("blackduck.api.token", "Black Duck API Token", "4.2.0", PropertyType.STRING, PropertyAuthority.NONE),
 
+    @HelpGroup(primary = GROUP_GENERAL, additional = { SEARCH_GROUP_BLACKDUCK, DEFAULT_HELP })
+    @HelpDescription("When set to true, user will be able to scan and discover license names in Black Duck")
+    BLACKDUCK_LICENSE_SEARCH("blackduck.license.search", "Black Duck License Search", "6.2.0", PropertyType.BOOLEAN, PropertyAuthority.NONE),
+
```

Comment:
I believe this only relates to the signature scanner. Perhaps this should go with the other signature scanner properties.

Reply:
The key may need to change as well.